bahmni-docker/docker-compose.yml

```diff
@@ -182,6 +182,7 @@ services:
       - bahmni-web
 
   reportsdb:
+    platform: linux/amd64
```

Comment:
Will this line work for other platforms also (or is this only for M1)? Or does this need to be commented and only uncommented if platform is M1?

Reply:
I think other platforms also use linux/amd64. Docker desktop defaults platform to linux/amd64, in M1 it is linux/arm64/v8